src/retasc/templates/extensions/dates.py

```diff
@@ -16,6 +20,18 @@ def weeks(value: int) -> timedelta:
     return timedelta(weeks=value)
 
 
+def hours(value: int) -> timedelta:
+    return timedelta(hours=value)
+
+
+def minutes(value: int) -> timedelta:
+    return timedelta(minutes=value)
+
+
+def seconds(value: int) -> timedelta:
```

Comment:
Could we define a factory to create `timedelta` wrapper functions? Then create function with this factory.

```
def _make_timedelta_factory(unit: str):
    def wrapper(value: int) -> timedelta:
        return timedelta(**{unit: value})
    wrapper.__name__ = unit
    return wrapper
```

e.g. hours = _make_timedelta_fn("hours")